Packs/Netmiko/Integrations/Netmiko/Netmiko.py

```diff
@@ -82,6 +88,8 @@ def test_command(client):
 
 def cmds_command(client, args):
 
+    interestingKeys = ("Hostname", "DateTimeUTC", "Command", "Output")
```

Comment:
let's use the `hdrs` variable below if it's exactly the same

Reply:
Done